README.md

```diff
@@ -30,145 +30,144 @@ Styleguide for low-fidelity paper prototyping and wireframing.
 - Images
 - Modal dialogs
 
-3.2. _MUST_. For **navigation bars**, when the bar is prefixed by a trademark or logo, include the word `LOGO` at the beggining of the bar.
+3.2. _MUST_ include the word `LOGO` at the beginning of **navigation bars** prefixed by a trademark or logo.
 
 ![](https://raw.githubusercontent.com/NOALVO/wireguide/master/img/32.PNG)
 
-3.3. _MUST_. **Modal dialogs** must include close button (`×`) at the top-right corner.
+3.3. _MUST_ include close buttons (`×`) at the top-right corner of **modal dialogs**.
 
 ![](https://raw.githubusercontent.com/NOALVO/wireguide/master/img/33.PNG)
 
-3.4. _MUST_. **Images** must be represented by a square with an x mark touching its corners.
+3.4. _MUST_ represent **images** by a square with an x mark touching its corners (`☒`).
 
 ![](https://raw.githubusercontent.com/NOALVO/wireguide/master/img/34.PNG)
 
-3.5. _SHOULD_. For **tables** and **general containers** that includes repeatable elements, draw only the first line of element and include three vertical dots at the middle of container.
+3.5. _SHOULD_ draw only the first line of a repeating elements, followed by a vertical ellipsis (`⋮`) at the middle of  the container for **tables** and **containers**.
 
 ![](https://raw.githubusercontent.com/NOALVO/wireguide/master/img/35.PNG)
 
-3.6. _SHOULD_. For **tables** and **general containers** that is supposed to be taller than wireframe bounds, DO NOT finish the container with a bottom side line, and apply a three vertcial dashes following each horizontal side of container.
+3.6. _SHOULD NOT_ draw bottom lines for **tables** and **containers** that are taller than the wireframe. Instead, apply three dashes (like an ellipsis) from each vertical line of the container's sides.
 
 ![](https://raw.githubusercontent.com/NOALVO/wireguide/master/img/36.PNG)
 
-3.7. _SHOULD_. For **tables** and **general containers** that is supposed to be wider than wireframe bounds, DO NOT finish the container with a bottom side line, and apply a three horizontal dashes following each vertical side of container.
+3.7. _SHOULD NOT_ draw right/left lines for **tables** and **containers** that are wider than the wireframe. Instead, apply three dashes (like an ellipsis) from each horizontal line of container's sides.
```

Comment:
1. Wider means that only right lines would exist. `Left` needs to be excluded of the sentence.

2. The same of 3.6 apply here. The mention of bounds need to be explicit. If it is not the right word, please suggest a synonym.

3. You put a `the` before `container's sides` at 3.6 but not here. Why?

Reply:
1. I misunderstood it. I thought it could be represented by either sides, but considering it's a non-established length, it would make no difference, after all.
2. Fixed.
3. Fixed.